### ISSUE TITLE
Terrier's stemmer and stopwords list

### ISSUE THREAD

**Issue by oussaidene** [terrier-org/pyterrier#366]
Hi,

I would like to know if there is a way to use terrier's stemmer and stopwords list outside of indexing?

Meaning, we'd be able to do something like this:
`stemmed = PorterStemmer(String)`
(and same thing for stopwords removal)

Thanks.

**Comment by cmacdonald** [comment]
Yes you can: 
```python

stemmer = pt.autoclass("org.terrier.terms.PorterStemmer")
stemmed = stemmer.stem("participated")
```

Similar for [Stopwords](http://terrier.org/docs/current/javadoc/org/terrier/terms/Stopwords.html) class.

**Comment by seanmacavaney** [comment]
To clarify, you need to construct the stemmer object, like so:

```
stemmer = pt.autoclass("org.terrier.terms.PorterStemmer")()
```

**Comment by seanmacavaney** [comment]
Perhaps it would be helpful to directly expose a `TerrierStemmer.stem()` method, [like this](https://github.com/terrier-org/pyterrier/compare/master...stemmer-stem)?

**Comment by cmacdonald** [comment]
clearly....

**Comment by cmacdonald** [comment]
Now addressed in #382
